### DISCUSSION TITLE
How to use local docker images?

### DISCUSSION THREAD

**Discussion by aybartsch** [firecow/gitlab-ci-local#1177] [Q&A]
I'm running gitlab-ci with docker in docker service. The script is processing a docker-compose file with some huge images inside. Unfortunately, these images are loaded every time anew. Would it be possible to setup or instruct gitlab-ci-local to make use of previously loaded images?

I'm also using docker desktop application on my Mac book which is having an internal image registry. Maybe, this one could be linked to gitlab-ci-local somehow?

**Comment by Nationalcat** [comment]
You can set local registry IP in `insecure-registry`, for more details see this PR #1201

**Comment by aybartsch** [comment]
@Nationalcat Thanks for this info. I'm using the GitlabCiLocal plugin in IntelliJ Idea and Docker Desktop application. I have enhanced the "gitlab-ci-local-env" file in my IDE accordingly. Could you please give me some advise how to setup the network cache with Docker Desktop?

Better question: How can I run the GitlabCiLocal plugin with the option "--network gitlab-ci-local-registry"? Could this parameter be added to "gitlab-ci-local-env" file?
